src/Symfony/Bundle/SecurityBundle/Resources/config/security_listeners.xml

```diff
@@ -45,6 +45,7 @@
             <argument type="service" id="logger" on-invalid="null" />
             <argument type="service" id="event_dispatcher" on-invalid="null" />
             <argument type="service" id="security.authentication.trust_resolver" />
+            <argument type="service" id="security.authentication.rememberme" on-invalid="null" />
```

Comment:
There is no such global service but one `RememberMeServices` per context (firewall). 
This argument needs to be wired from `SecurityExtension`. See `RememberMeFactory` which is responsible for the service registration.